Makefile

```diff
@@ -75,7 +75,7 @@ build: ## Build with default features enabled
 
 .PHONY: build-no-std
 build-no-std: ## Build without the standard library
-	cargo build --release --no-default-features --target wasm32-unknown-unknown
+	RUSTFLAGS='--cfg getrandom_backend="wasm_js"' cargo build --release --no-default-features --target wasm32-unknown-unknown
```

Comment:
I think this could also be set in a `config.toml` file that sets this var when the target is `wasm32-unknown-unknown` (as seen [here](https://github.com/uuid-rs/uuid/issues/792#issuecomment-2652095755)), might be cleaner but not sure